src/lib/containers/QueryVisualizationWrapper.tsx

```diff
@@ -12,6 +12,8 @@ import { NoContentPlaceholderType } from './table/NoContentPlaceholderType'
 import SearchResultsNotFound from './table/SearchResultsNotFound'
 import ThisTableIsEmpty from './table/TableIsEmpty'
 import { unCamelCase } from '../utils/functions/unCamelCase'
+import { ColumnType } from '../utils/synapseTypes'
+import { getDisplayValue } from '../utils/functions/getDataFromFromStorage'
```

Comment:
While this is imported for now, we can change it (e.g. to use react-query) later.